mne_bids/read.py

```diff
@@ -147,11 +147,12 @@ def _read_events(events_data, event_id, raw, verbose=None):
         verbose=verbose
     )
     all_dur = raw.annotations.duration
-if all_events.size == 0 and `_task_rest` not in raw.filenames[0]:
-            warn('No events found or provided. Please add annotations '
-                 'to the raw data, or provide the events_data and '
-                 'event_id parameters. If this is resting state data '
-                 'it is recommended to name the task "rest".')
+    if all_events.size == 0 and '_task_rest' not in raw.filenames[0] and \
+        verbose == True:
```

Comment:
We don't need the verbose switch or do we?
In my view we should always callt he logger and the logger **itself** must decide, based on the verbosity level, whether to print the message or not. It that not how logging in MNE works?

Reply:
Also this check wouldn't print a warning if verbose=='info' or verbose=='debug'...

Reply:
I'll leave this to @agramfort to assess :)